.github/workflows/reusable-e2e.yml

```diff
@@ -307,6 +307,9 @@ jobs:
         run: yarn docs build
         env:
           SITE_URL: ${{ secrets.SITE_URL }}
+          DOCSEARCH_DOCS_APP_ID: ${{ secrets.DOCSEARCH_DOCS_APP_ID }}
+          DOCSEARCH_DOCS_API_KEY: ${{ secrets.DOCSEARCH_DOCS_API_KEY }}
+          DOCSEARCH_DOCS_INDEX_NAME: ${{ secrets.DOCSEARCH_DOCS_INDEX_NAME }}
```

Comment:
Can you add these three to the secret list at the top of this file?

https://github.com/aws-amplify/amplify-ui/blob/2cd71b992bfec493c7b7c274ef6f86c63710cf7e/.github/workflows/reusable-e2e.yml#L13-L28

Reply:
Thanks! Done!

Reply:
Thanks! Done!